crates/wash-lib/src/parser/mod.rs

```diff
@@ -407,6 +407,10 @@ pub struct TinyGoConfig {
     pub tinygo_path: Option<PathBuf>,
     /// Whether to disable the `go generate` step in the build process. Defaults to false.
     pub disable_go_generate: bool,
+    /// The scheduler to use for the TinyGo build. Override the default scheduler (asyncify). Valid values are: none, tasks, asyncify.
+    pub scheduler: Option<String>,
```

Comment:
Should we check the values here? An enum would do well to restrain the values this can take on, these shouldn't change very often, right?

Reply:
Agreed, at the very least an enum with an `Unknown(String)` variant with a warning log is a good idea. I would only add that if we think that it's likely for TinyGo to add additional scheduler/gc options in a future release, otherwise validating + failing is a good idea

Reply:
added enum 👍